pkg/apis/externaldns/types_test.go

```diff
@@ -697,7 +684,7 @@ func TestParseFlagsCliFlagCobraParitySubset(t *testing.T) {
 	}
 
 	// Kingpin baseline without the hidden flag
-	baselineArgs := append([]string{}, args[1:]...)
+	baselineArgs := append([]string{}, args...)
 	cfgK := NewConfig()
 	require.NoError(t, cfgK.ParseFlags(baselineArgs))
```

Comment:
```suggestion
	cfgK := NewConfig()
	require.NoError(t, cfgK.ParseFlags(args))
```

Reply:
fixed

Something was not right with the suggestion, so pushed proposed fix
<img width="1133" height="601" alt="Screenshot 2025-12-19 at 12 57 25" src="https://github.com/user-attachments/assets/54d40af4-99fa-44c4-9c88-1556ed8a7974" />

Reply:
YEah. Resolved locally and pushed changes. Seems all good